tools/flux/tool-data/huggingface.loc.sample

```diff
@@ -0,0 +1,14 @@
+
+#This is a sample file distributed with Galaxy that is used to define huggingface
+#models, using three columns tab separated
+#(longer whitespace are TAB characters):
+#
+#The entries are as follows:
+#
+#<unique_build_id>	<display_name>	<folder_base_path>
+#
+#Your huggingface.loc file should include the huggingface cached model
+#folder you have stored.  For example:
+#
+#black-forest-labs/FLUX.1-dev	FLUX.1 [dev] is an open-weight, guidance-distilled model for non-commercial applications.	/path/to/huggingface/that-contains/hub/models--black-forest-labs--FLUX.1-dev
+#black-forest-labs/FLUX.1-schnell	FLUX.1 [schnell] is the fastest model, tailored for personal use.	/path/to/huggingface/that-contains/hub/models--black-forest-labs--FLUX.1-schnell
```

Comment:
we would need some more columns here, proposal:

* pipeline_tag (see this URL: https://huggingface.co/models?pipeline_tag=text-to-image they call it that way)
* domain (defined by the Galaxy community, e.g. image / sequence / text)
* free_tag (not idea yet, but could be freely used by an admin to specify more filter options)
* version (if available)

Have just talked with @anuprulez about this, he would like to use the same data table ... lets talk about this in the next days.